lib/exercism/user.rb

```diff
@@ -102,7 +103,30 @@ def owns?(submission)
     self == submission.user
   end
 
-  private
+  def increment_five_a_day
+    if FiveADayCount.exists?(["user_id = ? and day = ?", self.id, Date.today])
```

Comment:
This is not important here, but since there's a relationship between the user and the counts, you can say `five_a_day_counts.where(:day, Date.today).exists?`